ui/src/components/SampleGrid/TaskItem.jsx

```diff
@@ -174,23 +174,23 @@ export class TaskItem extends React.Component {
           <div className="row" style={{ paddingTop: '0.5em' }}>
             <span className="col-sm-4">
               <b>Quality Indictor: </b>
-              <img ref="fimage" alt="First" src={qIndUrl} width="90%" />
```

Comment:
Couldn't find any use of these refs, but they could be hiding... Please let me know if that's the case.